### ISSUE TITLE
Log spam after chatlog fills up - 202.5.6

### ISSUE THREAD

**Issue by Fluboxer** [mrbuilder1961/WheresMyChatHistory#130]
### Description
Basically it spams logs with stuff like "[ChatLog.addMessage] ChatLog message capacity has been reached, ignoring message '<Fluboxer> 123'" making reading those more miserable than it should be 
Also chat history can't keep itself and gets lost when you reboot game - after file is full

### How to reproduce
I guess all you have to do is to fill limit?

### Expected result
I expected it to erase old messages to fit new ones, not to spam warning for EVERY string in chat 

### Log file link
https://pastebin.com/AiusGjLg

### Specs and Details
- Astra Linux (5.15.0)
- Mod version is 202.5.6
- Fabric 0.14.22; Fabric API 0.90.4
- YACL 3.3.0; ModMenu 8.0.0
- I disabled other mods to make sure that this issue persists 

Optional but suggested information:
- `chatpatches.json` file:
```
{
  "time": false,
  "timeDate": "HH:mm:ss",
  "timeFormat": "[$]",
  "timeColor": 16733695,
  "hover": false,
  "hoverDate": "MM/dd/yyyy",
  "hoverFormat": "$",
  "hoverColor": 16777215,
  "counter": true,
  "counterFormat": "\u00268[\u00267x\u0026r$\u00268]",
  "counterColor": 16777045,
  "counterCompact": false,
  "counterCompactDistance": 0,
  "boundary": true,
  "boundaryFormat": "\u00268[\u0026r$\u00268]",
  "boundaryColor": 5636095,
  "chatLog": true,
  "chatHidePacket": true,
  "chatWidth": 0,
  "chatMaxMessages": 16384,
  "chatNameFormat": "\u003c$\u003e",
  "shiftChat": 10,
  "messageDrafting": false,
  "searchDrafting": true,
  "hideSearchButton": true,
  "vanillaClearing": false,
  "copyColor": 5636095,
  "copyReplyFormat": "/msg $ "
}
```
- `chatlog.json` file: [chatlog.json](https://github.com/mrbuilder1961/ChatPatches/files/13230001/chatlog.json)

### Additional context
Noticed it when were checking logs for different issue; it also seems to affect chat history that DO work until you restart game

**Comment by mrbuilder1961** [comment]
fix for this will be out soon, in a few days max. however, i did notice in your latest.log there were no messages about quitting the game, did you Alt+F4 out of Minecraft or something? your log ends abruptly, and the chat log is supposed to notify when it has saved which is why i am curious.

**Comment by Fluboxer** [comment]
I close it normally. May be a linux thing or my distro shenanigans in particular

**Comment by mrbuilder1961** [comment]
Do any of your logs have something similar to this at the end?
```log
[18:40:28] [Server thread/INFO]: Saving and pausing game...
[18:40:28] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New World]'/minecraft:overworld
[18:40:28] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New World]'/minecraft:the_nether
[18:40:28] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New World]'/minecraft:the_end
[18:40:28] [Server thread/INFO]: MechanicalArcane lost connection: Disconnected
[18:40:28] [Server thread/INFO]: MechanicalArcane left the game
[18:40:28] [Server thread/INFO]: Stopping singleplayer server as player logged out
[18:40:28] [Server thread/INFO]: Stopping server
[18:40:28] [Server thread/INFO]: Saving players
[18:40:28] [Server thread/INFO]: Saving worlds
[18:40:29] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New World]'/minecraft:overworld
[18:40:32] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New World]'/minecraft:the_nether
[18:40:32] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New World]'/minecraft:the_end
[18:40:32] [Server thread/INFO]: ThreadedAnvilChunkStorage (New World): All chunks are saved
[18:40:32] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM-1): All chunks are saved
[18:40:32] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM1): All chunks are saved
[18:40:32] [Server thread/INFO]: ThreadedAnvilChunkStorage: All dimensions are saved
[18:40:33] [Render thread/INFO]: Stopping!
```
I'm not sure if this is a problem with Chat Patches or a problem with how your game exits.

**Comment by Fluboxer** [comment]
Those were here when I did tests and some of them involved going in single-player
[2023-11-01-3.log](https://github.com/mrbuilder1961/ChatPatches/files/13259457/2023-11-01-3.log)

**Comment by mrbuilder1961** [comment]
that's interesting, the chat log saved in that case. this must be something different, but i haven't had this issue so im not sure i could fix it at the moment. as for this issue, it should be fixed in `202.5.7`. please reopen it if the issue hasn't fixed, thanks for the report!
